changelog.d/20240208_180713_30907815+rjmello_pasl_htex_encryption.rst

```diff
@@ -0,0 +1,14 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Upgraded Parsl to version ``2024.02.05`` to enable encryption for the ``GlobusComputeEngine``.
+  Under the hood, Parsl uses CurveZMQ to encrypt all communication channels between the engine
+  and related nodes.
+
+  We enable encryption by default, but users can disable it by setting the ``encrypted``
+  initialization argument for the ``GlobusComputeEngine`` to ``False``.
```

Comment:
Maybe it's just me, but it isn't obvious how to set this in config.yaml.  ie. which level to put ``encrypted: true``

Should we add an example, or is this a fairly simple thing for most people?

Reply:
Good call; my head was still in Parsl-mode when writing this up. I've updated to clarify where to put this variable in the YAML config and included an example.